internal/sdkv2provider/resource_cloudflare_teams_accounts.go

```diff
@@ -100,12 +100,15 @@ func resourceCloudflareTeamsAccountRead(ctx context.Context, d *schema.ResourceD
 	}
 
 	payloadLogSettings, err := client.GetDLPPayloadLogSettings(ctx, cloudflare.AccountIdentifier(accountID), cloudflare.GetDLPPayloadLogSettingsParams{})
-	if err != nil {
-		return diag.FromErr(fmt.Errorf("error finding DLP Account config %q: %w", d.Id(), err))
-	}
-
-	if err := d.Set("payload_log", flattenPayloadLogSettings(&payloadLogSettings)); err != nil {
-		return diag.FromErr(fmt.Errorf("error parsing payload log settings: %w", err))
+	if err == nil {
+		if err := d.Set("payload_log", flattenPayloadLogSettings(&payloadLogSettings)); err != nil {
+			return diag.FromErr(fmt.Errorf("error parsing payload log settings: %w", err))
+		}
+	} else {
+		// A 404 error is acceptable here
+		if !strings.Contains(err.Error(), "not found") {
```

Comment:
instead of a string check, let's check for the specific request error. you can see an example of this at https://github.com/cloudflare/terraform-provider-cloudflare/blob/0f9af804800f8ea3be14455a683756d0d1336bb0/internal/sdkv2provider/resource_cloudflare_account.go#L69-L70.